### PR TITLE
Add Nodes listing

### PR DESCRIPTION
**Pull request by shinji62**
Added Nodes listing command 
Added Nodes listing in the cluster view
Screen Shot 2023-03-17 at 11 10 46
Screen Shot 2023-03-17 at 11 11 03

### CODE REVIEW

**Review by posriniv — APPROVED** [review]
Looks good to me

**Review by cdavid — APPROVED** [review]

File: cmd/cluster/cluster.go [hunk 1]
```diff
@@ -42,4 +43,6 @@ func init() {
 	ClusterCmd.AddCommand(readreplica.ReadReplicaCmd)
 	readreplica.ReadReplicaCmd.PersistentFlags().StringVarP(&readreplica.ClusterName, "cluster-name", "c", "", "[REQUIRED] The name of the cluster.")
 	readreplica.ReadReplicaCmd.MarkPersistentFlagRequired("cluster-name")
+
+	ClusterCmd.AddCommand(node.NodeCmd)
```
Comment (cdavid) [inline cmd/cluster/cluster.go:47]: Can you please add this under a feature flag?

Comment (cdavid) [reply cmd/cluster/cluster.go:47]: https://yugabyte.slack.com/archives/C04G8JB4GSD/p1678852877456819?thread_ts=1678842619.217429&cid=C04G8JB4GSD

Comment (shinji62) [reply cmd/cluster/cluster.go:47]: This command just list the nodes. that's all, I don't see why we should put behind flag.

The others command(start/stop a node) when implemented should be under feature flag.

**Pull request merged**